src/main/java/seedu/address/logic/Messages.java

```diff
@@ -11,6 +16,20 @@ public class Messages {
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_DUPLICATE_FIELDS =
+                "Multiple values specified for the following single-valued field(s): ";
+
+    /**
+     * Generates a formatted error message indicating the duplicate prefixes.
```

Comment:
`a formatted error message` -> `an error message` (as I mentioned in my previous comment)
While you are at it, `Returns`  may be better than `Generates`.